stylus/ui-components/button.styl

```diff
@@ -34,6 +34,11 @@ $button--more
     width      em(46px)
     background embedurl("../assets/icons/ui/icon-dots.svg") center center no-repeat
 
+$button--close
+    width      em(46px)
+    background embedurl("../assets/icons/ui/icon-cross.svg") center center no-repeat
```

Comment:
The linter that checks Stylus syntax has been merged yesterday, so to avoid future errors you should change double quote to single quote (and I'm not sure I wrote this in the guidelines yet, so I'm gonna do it right after this review)